src/Pages/LeaderBoard.js

```diff
@@ -58,12 +131,12 @@ function LeaderBoard() {
                                       </tr>
                                     </thead>
                                     <tbody id="protocols-tbody" className="divide-y divide-gray-200">
-                                      {
-                                        data
+                                      {  
+                                        allProtocols
```

Comment:
If you're interested in app performance, these kinds of operations are a good use case for the useMemo hook or some sort of memoization as it can be quite an expensive operation to perform on every rerender. No issue doing it directly here though other than performance implications.